src/autoscaler/db/sqldb/appmetric_sqldb_test.go

```diff
@@ -247,4 +248,78 @@ var _ = Describe("AppMetricSQLDB", func() {
 
 	})
 
+	Describe("PruneAppMetrics", func() {
```

Comment:
You mean failure test case for pruneMetrics() ?
Here in the fake database is not being used, so we followed the approach of other test cases across.